engine/src/main/java/org/terasology/world/internal/WorldInfo.java

```diff
@@ -82,8 +82,7 @@ public void setWorldGenerator(SimpleUri worldGenerator) {
     }
 
     @Override
-    public String toString()
-    {
+    public String toString(){
```

Comment:
You also have to add a blank space between `()` and `{` as you did in other methods.